test/services/lessons/addMaterialUnit.test.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert');
+const chai = require('chai');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['chai' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4977295)